packages/main/src/plugin/statusbar/statusbar-item.ts

```diff
@@ -92,6 +93,15 @@ export class StatusBarItemImpl implements StatusBarItem {
     this.update();
   }
 
+  public get notify(): boolean {
```

Comment:
I would not introduce a notify option that will lead to a lot of blinks everywhere.
We can provide an icon like warning icon

we're following most of the VS Code API and there is no notify so I think we can do without it

Reply:
Thanks